app/scripts/components/analysis/results/chart-card.tsx

```diff
@@ -117,8 +118,9 @@ const getNoDownloadReason = ({ status, data }: TimeseriesData) => {
  * @returns Internal path for Link
  */
 const getDatasetOverviewPath = (layerId: string) => {
+  const allDatasetsProps = getAllDatasetsProps(veda_faux_module_datasets);
   const dataset = allDatasetsProps.find((d) =>
-    d.layers.find((l) => l.id === layerId)
+    d?.layers.find((l) => l.id === layerId)
```

Comment:
any reason that optional chain is needed here?

Reply:
I dont remember why i added that optional chain... but its safer with 😄